src/core/qgsvectorfilewriter.cpp

```diff
@@ -1551,6 +1565,10 @@ bool QgsVectorFileWriter::driverMetadata( const QString& driverName, QgsVectorFi
   return false;
 }
 
+OGRwkbGeometryType QgsVectorFileWriter::ogrTypeFromWkbType( QgsWKBTypes::Type type )
+{
+  return static_cast<OGRwkbGeometryType>( type );
```

Comment:
I don't know, but things like this was why I introduced this method instead of using straight casting.